src/drive/web/modules/selectors.js

```diff
@@ -28,6 +28,14 @@ export const getDisplayedFolder = rootState => {
   return null
 }
 
+export const getParentFolder = (rootState, parentFolderId) => {
+  if (parentFolderId) {
+    const doc = getDocumentFromState(rootState, 'io.cozy.files', parentFolderId)
```

Comment:
This is useful only if the parent has been loaded first. Is it always the case? What happen if we're accessing a subfolder directly?

Reply:
> What happen if we're accessing a subfolder directly?

You're right, in that case we don't show a parent breadcrumb. This is where we would need a dedicated network request for the breadcrumb.